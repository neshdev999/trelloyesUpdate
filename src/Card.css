.cardContainer {
    display: flex;
    justify-content: center;
    align-items: flex-start;
    flex-direction: column;
    background-color: #ffffff;
    margin: 5px;
    padding: 10px;
    border-radius: 5px;
}

.cardContentItem {
    flex: 1;
    margin: 0px;
    padding: 0px;
}

.cardContentFirstItems {
    display: flex;
    flex-direction: row;
    justify-content: flex-start;
    align-items: center;
}

.cardContentFirstItemsPeta {
    flex: 1;
    padding: 0px;
    margin: 0px;
}

.cardHeader {
    font-size: 16px;
    white-space: nowrap;
    padding-right: 15px;
    padding-left: 0px;
    padding-top: 0px;
    padding-bottom: 0px;
}

.cardContentSecondItem {
    /* margin: 5px; */
    font-size: 12px;
}