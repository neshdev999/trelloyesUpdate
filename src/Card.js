import React from 'react';
import './Card.css'

class Card extends React.Component{    
    render(){
        let currentCardId = this.props.id;
        return(
            <div className="cardContainer">
                <div className="cardContentItem cardContentFirstItems">
                    <div className="cardContentFirstItemsPeta">
                        <h3 className="cardHeader">{this.props.cardStore.allCards[currentCardId]['title']}</h3>
                    </div>
                    <div className="cardContentFirstItemsPeta">
                        <button className="cardButton">delete</button>
                    </div>
                </div>
                <div className="cardContentItem">
                    <div className="cardContentSecondItem">
                        <p className="cardContent">{this.props.cardStore.allCards[currentCardId]['content']}</p>
                    </div>             
                </div>
            </div>
        )
    }
}

export default Card;