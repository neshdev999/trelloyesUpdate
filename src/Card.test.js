import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import Store from './store';
import renderer from 'react-test-renderer';

/* Smoke Tests */
describe('First List cards renders correctly', ()=>{

    it('card renders without crashing', () =>{
        const div = document.createElement('div');
        const currentStore = Store;
        const currentCardIds = ['a', 'b', 'e', 'f', 'g', 'j', 'l', 'm'];

        currentCardIds.map(ites => {
            ReactDOM.render(<Card id={ites} cardStore={currentStore} />, div);
        });    
        ReactDOM.unmountComponentAtNode(div);
    });
});

describe('Second List cards renders correctly', ()=>{

    it('card renders without crashing', () =>{
        const div = document.createElement('div');
        const item = 'a';
        const currentStore = Store;
        const currentCardIds = ['b', 'c', 'd', 'f', 'h', 'i', 'k'];

        currentCardIds.map(ites => {
            ReactDOM.render(<Card id={ites} cardStore={currentStore} />, div);
        });    
        ReactDOM.unmountComponentAtNode(div);
    });
});


describe('Third List cards renders correctly', ()=>{

    it('card renders without crashing', () =>{
        const div = document.createElement('div');
        const item = 'a';
        const currentStore = Store;
        const currentCardIds = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm'];

        currentCardIds.map(ites => {
            ReactDOM.render(<Card id={ites} cardStore={currentStore} />, div);
        });    
        ReactDOM.unmountComponentAtNode(div);
    });
});


describe('Fourth List cards renders correctly', ()=>{

    it('card renders without crashing', () =>{
        const div = document.createElement('div');
        const item = 'a';
        const currentStore = Store;
        const currentCardIds = ['l', 'm'];

        currentCardIds.map(ites => {
            ReactDOM.render(<Card id={ites} cardStore={currentStore} />, div);
        });    
        ReactDOM.unmountComponentAtNode(div);
    });
});


/* Snapshot Tests */



describe('First List cards UI renders as expected', ()=>{
    it('renders the UI as expected', () =>{
        const currentStore = Store;
        const currentCardIds = ['a', 'b', 'e', 'f', 'g', 'j', 'l', 'm'];
        
    
        currentCardIds.map(ites => {
            const tree = renderer.create(<Card id={ites} cardStore={currentStore} />).toJSON();
            expect(tree).toMatchSnapshot();
        }); 
    });
});

describe('Second List cards UI renders as expected', ()=>{
    it('renders the UI as expected', () =>{
        const currentStore = Store;
        const currentCardIds = ['b', 'c', 'd', 'f', 'h', 'i', 'k'];
        
    
        currentCardIds.map(ites => {
            const tree = renderer.create(<Card id={ites} cardStore={currentStore} />).toJSON();
            expect(tree).toMatchSnapshot();
        }); 
    });
});

describe('Third List cards UI renders as expected', ()=>{
    it('renders the UI as expected', () =>{
        const currentStore = Store;
        const currentCardIds = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm'];
        
    
        currentCardIds.map(ites => {
            const tree = renderer.create(<Card id={ites} cardStore={currentStore} />).toJSON();
            expect(tree).toMatchSnapshot();
        }); 
    });
});

describe('Fourth List cards UI renders as expected', ()=>{
    it('renders the UI as expected', () =>{
        const currentStore = Store;
        const currentCardIds = ['l', 'm'];
        
    
        currentCardIds.map(ites => {
            const tree = renderer.create(<Card id={ites} cardStore={currentStore} />).toJSON();
            expect(tree).toMatchSnapshot();
        }); 
    });
});
