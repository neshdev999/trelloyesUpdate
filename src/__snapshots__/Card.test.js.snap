// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First List cards UI renders as expected renders the UI as expected 1`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        First card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`First List cards UI renders as expected renders the UI as expected 2`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Second card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`First List cards UI renders as expected renders the UI as expected 3`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Fifth card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`First List cards UI renders as expected renders the UI as expected 4`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Sixth card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`First List cards UI renders as expected renders the UI as expected 5`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Seventh card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`First List cards UI renders as expected renders the UI as expected 6`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Tenth card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`First List cards UI renders as expected renders the UI as expected 7`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Twelfth card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`First List cards UI renders as expected renders the UI as expected 8`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Thirteenth card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`Fourth List cards UI renders as expected renders the UI as expected 1`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Twelfth card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`Fourth List cards UI renders as expected renders the UI as expected 2`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Thirteenth card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`Second List cards UI renders as expected renders the UI as expected 1`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Second card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`Second List cards UI renders as expected renders the UI as expected 2`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Third card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`Second List cards UI renders as expected renders the UI as expected 3`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Fourth card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`Second List cards UI renders as expected renders the UI as expected 4`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Sixth card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`Second List cards UI renders as expected renders the UI as expected 5`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Eighth card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`Second List cards UI renders as expected renders the UI as expected 6`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Ninth card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`Second List cards UI renders as expected renders the UI as expected 7`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Eleventh card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`Third List cards UI renders as expected renders the UI as expected 1`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        First card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`Third List cards UI renders as expected renders the UI as expected 2`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Second card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`Third List cards UI renders as expected renders the UI as expected 3`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Third card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`Third List cards UI renders as expected renders the UI as expected 4`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Fourth card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`Third List cards UI renders as expected renders the UI as expected 5`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Fifth card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`Third List cards UI renders as expected renders the UI as expected 6`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Sixth card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`Third List cards UI renders as expected renders the UI as expected 7`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Seventh card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`Third List cards UI renders as expected renders the UI as expected 8`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Eighth card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`Third List cards UI renders as expected renders the UI as expected 9`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Ninth card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`Third List cards UI renders as expected renders the UI as expected 10`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Tenth card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`Third List cards UI renders as expected renders the UI as expected 11`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Eleventh card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`Third List cards UI renders as expected renders the UI as expected 12`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Twelfth card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;

exports[`Third List cards UI renders as expected renders the UI as expected 13`] = `
<div
  className="cardContainer"
>
  <div
    className="cardContentItem cardContentFirstItems"
  >
    <div
      className="cardContentFirstItemsPeta"
    >
      <h3
        className="cardHeader"
      >
        Thirteenth card
      </h3>
    </div>
    <div
      className="cardContentFirstItemsPeta"
    >
      <button
        className="cardButton"
      >
        delete
      </button>
    </div>
  </div>
  <div
    className="cardContentItem"
  >
    <div
      className="cardContentSecondItem"
    >
      <p
        className="cardContent"
      >
        lorem ipsum
      </p>
    </div>
  </div>
</div>
`;
